src/esp/scene/SceneNode.h

```diff
@@ -75,6 +76,9 @@ class SceneNode : public MagnumObject {
   //! set local bounding box for meshes stored at this node
   void setMeshBB(Magnum::Range3D meshBB) { meshBB_ = meshBB; };
 
+  //! set the global bounding box for mesh stored in this node
+  void setAbsoluteAABB(Magnum::Range3D aabb) { aabb_ = aabb; };
```

Comment:
setter but no getter? How is this used? 

Oh. Its protected. private with a getter is generally preferred. Raw ivar access is fragile.

Reply:
Yes, I actually have the getter in the branch "culling", which would be the next PR after this one committed in.

It looks like this:

```CPP
  Corrade::Containers::Optional<Magnum::Range3D> getAbsoluteAABB() {
    return aabb_;
  }
```

I can move it to this PR.